doc/sphinxext/contrib_avatars.py

```diff
@@ -34,7 +34,7 @@ def generate_contrib_avatars(app, config):
         driver.get(f"file://{infile}")
         wait = WebDriverWait(driver, 20)
         wait.until(lambda d: d.find_element(by=By.ID, value="contributor-avatars"))
-        body = driver.find_element(by=By.TAG_NAME, value="body").get_attribute(
+        body = driver.find_element(by=By.TAG_NAME, value="body").get_dom_attribute(
```

Comment:
... but maybe I did it wrong, I didn't look at the output yet...

Reply:
ah OK. I was just going by the deprecation message and hoping it would work, planning to follow-up if it didn't. But if you're already addressing elsewhere I'll close this.